packages/content/lib/index.js

```diff
@@ -208,9 +208,15 @@ module.exports = async function (moduleOptions) {
       fileName: 'content/nuxt-content.dev.vue',
       src: join(__dirname, '../templates/nuxt-content.dev.vue'),
       options: {
-        apiPrefix: options.apiPrefixWithBase
+        apiPrefix: options.apiPrefixWithBase,
+        editor: options.editor
       }
     })
+    // Add dev editor component
+    this.addTemplate({
+      fileName: 'content/editor.vue',
+      src: join(__dirname, '../templates/editor.vue')
```

Comment:
```suggestion
      src: join(__dirname, '..', 'templates', 'editor.vue')
```

To respect Windows backslahes